### PR TITLE
[All] (Hub page) Accessibility and SEO updates

### PR DESCRIPTION
**Pull request by alison-mk**
Accessibility and SEO updates per internal review.

### CODE REVIEW

File: docs/index.yml [hunk 1]
```diff
@@ -40,11 +40,11 @@ highlightedContent:
...
     # Card 6
     - itemType: how-to-guide
-      title: Beginner's guide
+      title: Office Add-ins Beginner's guide
       url: overview/learning-path-beginner.md
     # Card 7
     - itemType: reference
-      title: API reference documentation
+      title: Office JavaScript API reference documentation
       url: https://docs.microsoft.com/javascript/api/overview
```
Comment (ElizabethSamuel-MSFT) [inline docs/index.yml:48]: Should be able to use a relative path here
```suggestion
      url: /javascript/api/overview
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
One small suggestion to use relative path for other docset

File: docs/index.yml (hunk 1 above)
Comment (alison-mk) [reply docs/index.yml:48]: There are a number of docs.microsoft.com links in this file, should I make this same change for all of them? 
For example: https://docs.microsoft.com/office/dev/store/ -> /office/dev/store/

Comment (ElizabethSamuel-MSFT) [reply docs/index.yml:48]: Let's confirm that the link still works then update the rest

File: docs/index.yml [hunk 2]
```diff
@@ -29,11 +29,11 @@ highlightedContent:
       # Card 3
     - itemType: learn
       title: "Microsoft Learn: Introduction to Office client customization with add-ins"
-      url: https://docs.microsoft.com/learn/modules/intro-office-add-ins/
```
Comment (ElizabethSamuel-MSFT) [inline docs/index.yml:32]: I would remove the slashes from the ends of these links

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
just one suggestion

**Pull request merged**